students/david_baylor/lesson2/fizzBuzz.py

```diff
@@ -0,0 +1,23 @@
+"""
+fizzBuzz.py
+by David Baylor
+uses python 3
+
+loops through numbers 1 to 100 and for multipuls of 3 it prints fizz, for multipuls of 5 it prints buzz and for multipuls of 3 and
+5 it prints FizzBuzz.
+"""
+
+def detFactor(n):
```

Comment:
Function names should be clear and snake_case.  For example def determin_factor(n):